### DISCUSSION TITLE
k3s-airgap-images-amd64.tar Forbidden

### DISCUSSION THREAD

**Discussion by drakon660** [rancher-sandbox/rancher-desktop#1527] [Q&A]
How can I get source of this file and any other that Rancher Desktop is downloading to proper open connection in business environment.

**Comment by jandubois** [comment]
Not sure where your "Forbidden" error comes from.

The k3s binary and airgap images are loaded from the Github releases at https://github.com/k3s-io/k3s/releases.

**Comment by drakon660** [comment]
Error comes from blocked network, would be great to have this information in debug logs.

**Comment by drakon660** [comment]
Thanks a lot.
